.github/workflows/release.yml

```
@@ -0,0 +1,105 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - 'v*'
+
+jobs:
+  build-and-release:
+    runs-on: macos-latest
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)